e2e/fixtures/r/foo.R

```diff
@@ -0,0 +1,20 @@
+# @octoLinkerResolve(https://github.com/r-lib/rlang)
```

Comment:
@stefanbuck are these case sensitive?

Reply:
Yes, good spot

Reply:
@edavidaja please update all OctoLinker annotations to ensure E2E tests are running

```diff
- @OctoLinkerResolve
+ @OctoLinkerResolve
```

Reply:
fixed